packages/shared/routes/onboarding/views/app-setup/views/LanguageAndAppearanceView.svelte

```diff
@@ -36,11 +37,14 @@
     }
 
     function handleContinueClick(): void {
-        $appRouter.next()
+        if ($appStage !== AppStage.PROD) {
```

Comment:
Minor security thing: Allowlists >>> Blocklists!

```suggestion
        if ($appStage === AppStage.ALPHA || $appStage === AppStage.BETA) {
```